TestUtils.ts

```diff
@@ -55,6 +55,7 @@ const DEFAULT_GLOBAL_CONFIG: Config.GlobalConfig = {
   testNamePattern: '',
   testPathPattern: '',
   testResultsProcessor: null,
+  timeout: 5000,
```

Comment:
can we call it `testTimeout` instead of `timeout`? I think just setting `timeout` seems like it affectes the whole run, not just a single test

Reply:
ok I will change it